.github/workflows/sec-scan.yml

```diff
@@ -0,0 +1,60 @@
+##
+# This action runs trivy container scanning and anchore grype vulnerability
+# scanner for docker images and packages.
+##
+
+name: sec-scan
+
+on:
+  repository_dispatch:
+    types: [sec-scan-trigger]
+
+jobs:
+  trivy_scan_image:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        image:
+          - farmer
+          - node
+          - bootstrap-node
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8 # @v3.1.0
+
+      - name: Run Trivy vulnerability scanner on image
+        uses: aquasecurity/trivy-action@d43c1f16c00cfd3978dde6c07f4bbcf9eb6993ca # @v0.16.1
+        with:
+          image-ref: ghcr.io/${{ github.repository_owner }}/${{ matrix.image }}:${{ github.event.client_payload.tag }}
```

Comment:
Why using tag (whose contents can change) rather than image hash?

Reply:
True, omitted that and just sending the hash